packages/core/core/src/Transformation.js

```diff
@@ -537,13 +537,15 @@ async function runTransformer(
 
   const resolve = async (from: FilePath, to: string): Promise<FilePath> => {
     return nullthrows(
-      await pipeline.resolverRunner.resolve(
-        createDependency({
-          env: asset.value.env,
-          moduleSpecifier: to,
-          sourcePath: from,
-        }),
-      ),
+      (
+        await pipeline.resolverRunner.resolve(
+          createDependency({
+            env: asset.value.env,
+            moduleSpecifier: to,
+            sourcePath: from,
+          }),
+        )
+      ).assetGroup,
```

Comment:
This call shouldn't discard the files to be watched either..

Reply:
I guess that would make sense?